spec/PasswordPolicy.spec.js

```diff
@@ -29,7 +29,11 @@ describe("Password Policy: ", () => {
       user.set('email', 'user@parse.com');
       return user.signUp();
     }).then(user => {
-      Parse.User.requestPasswordReset("user@parse.com");
+      Parse.User.requestPasswordReset('user@parse.com').catch((err) => {
```

Comment:
@bhaskaryasa 
This should just be a simple 
`return Parse.User.requestPasswordReset("user@parse.com");`

You already have a catch handler at the bottom. Any errors will be caught by the catch at the bottom of the test case.
Pls let me know if I am missing something here.